README.md

```diff
@@ -7,50 +7,36 @@
 [![License: MIT](https://img.shields.io/github/license/microsoft/semantic-kernel)](https://github.com/microsoft/semantic-kernel/blob/main/LICENSE)
 [![Discord](https://img.shields.io/discord/1063152441819942922?label=Discord&logo=discord&logoColor=white&color=d82679)](https://aka.ms/SKDiscord)
 
-> ℹ️ **NOTE**: This project is just like AI and will evolve quickly.
-> We invite you to join us in developing the Semantic Kernel together!
-> Please contribute by
-> using GitHub [Discussions](https://github.com/microsoft/semantic-kernel/discussions),
-> opening GitHub [Issues](https://github.com/microsoft/semantic-kernel/issues/new/choose),
-> sending us [PRs](https://github.com/microsoft/semantic-kernel/pulls),
-> joining our [Discord community](https://aka.ms/SKDiscord).
-
-**Semantic Kernel (SK)** is a lightweight SDK enabling integration of AI Large
-Language Models (LLMs) with conventional programming languages. The SK extensible
-programming model combines natural language **semantic functions**, traditional
-code **native functions**, and **embeddings-based memory** unlocking new potential
-and adding value to applications with AI.
-
-SK supports
-[prompt templating](https://learn.microsoft.com/en-us/semantic-kernel/prompt-engineering/prompt-template-syntax), function
-chaining,
-[vectorized memory](https://learn.microsoft.com/en-us/semantic-kernel/memories/embeddings), and
-[intelligent planning](https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/planner)
-capabilities out of the box.
-
-Semantic Kernel supports and encapsulates several design patterns from the latest
-in AI research, such that developers can infuse their applications with [plugins](https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/plugins) like [prompt
-chaining](https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/chaining-functions), recursive reasoning, summarization, zero/few-shot learning, contextual
-memory, long-term memory, [embeddings](https://learn.microsoft.com/en-us/semantic-kernel/memories/embeddings), semantic indexing,
-[planning](https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/planner), retrieval-augmented generation and accessing external
-knowledge stores as well as your own data.
-
-By joining the SK community, you can build AI-first apps faster and have a front-row
-peek at how the SDK is being built. SK has been released as open-source so that more
-pioneering developers can join us in crafting the future of this landmark moment
-in the history of computing.
-
-## Get Started with Semantic Kernel ⚡
-
-Semantic Kernel is available to explore AI and build apps with C#, Python and Java:
+[Semantic Kernel](https://learn.microsoft.com/en-us/semantic-kernel/overview/)
```

Comment:
I don't see any mention of the VS Code extension... this should be promoted here with a screenshot